gapic-generator-java/DEVELOPMENT.md

```diff
@@ -109,16 +109,22 @@ To generate a production GAPIC API:
         strip_prefix = "gapic-generator-java-%s" % _gapic_generator_java_version,
         urls = ["https://github.com/googleapis/gapic-generator-java/archive/v%s.zip" % _gapic_generator_java_version],
     )
+    
+    # gax-java is part of gapic-generator-java repository
```

Comment:
Is this required? Tomo and I actually just discussed today, since gax is already included in the gapic-generator-java jar, as long as we built a new jar with local gax changes, we shouldn't need this step.

Reply:
I see the problem now, yes there are gax references in `java_gapic_library` that is used for generating the gradle files. Now that gax repo is pointed to `gapic-generator-java/gax-java`, we have to point it to local folder otherwise it will complaint something about "SNAPSHOR version does not exist". Thanks for fixing it!
CC: @suztomo I know you might be working on improving this part, this is one more side effect of the gradle files generation.